### PR TITLE
SelectBox: Add aria-describedby for enum descriptions. Fixes: #57710

### LINKED ISSUE #57710: Settings:SelectBox: Add aria-describedby for extended descriptions
New enum/select box detailed descriptions should be read by reader.
Add describedby attribute handling for details provider within selectbox.

### PR DESCRIPTION
**Pull request by cleidigh**
- Add extended descriptions for ARIA

Fixes: #57710

### CODE REVIEW

File: src/vs/base/browser/ui/selectBox/selectBoxCustom.ts [hunk 1]
```diff
@@ -57,7 +60,14 @@ class SelectListRenderer implements IRenderer<ISelectOptionItem, ISelectListTemp
 		const optionDisabled = (<ISelectOptionItem>element).optionDisabled;
 
 		data.optionText.textContent = optionText;
-		data.root.setAttribute('aria-label', nls.localize('selectAriaOption', "{0}", optionText));
+		data.root.setAttribute('aria-label', nls.localize('selectAriaOption', "{0}", optionText) + ',.');
```
Comment (roblourens) [inline src/vs/base/browser/ui/selectBox/selectBoxCustom.ts]: Hey @cleidigh why the `,.` here?

Comment (cleidigh) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts]: both , and .  will cause most screen readers to add a pause
in this case I found it really important to separate with a pause the options from the detailed description

with a short option like 'off'  it was almost completely missed by the subsequent description.

I was planning to add this to the other controls unless you have an objection

Comment (roblourens) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts]: Sure, that makes sense. Could you add a comment in the code that explains that?

Comment (cleidigh) [reply src/vs/base/browser/ui/selectBox/selectBoxCustom.ts]: sure, I'll do it along with the gear icon keyboard shortcut readout
...and comment

**Pull request merged**